client/js/_author/components/assessments/_edit_assessment.jsx

```diff
@@ -99,10 +114,31 @@ export class NewAssessment extends React.Component {
     return { ...this.props.assessment, ...this.state.assessment };
   }
 
+  assignedAssessment(published) {
```

Comment:
i think that a more discriptive name is needed for this. The name doesnt really describe what this method does. like `editOrPublishAssignedAssessment`